frontend/src/book-read/components/protected-review.wrapper.jsx

```diff
@@ -0,0 +1,24 @@
+import ProtectedWrapper from '../../auth/components/protected.wrapper';
```

Comment:
I write this only because you will have to do another commit. So... Please, re-export ProtectedWrapper in index.js of auth module and change import at routes.js file across the project.